src/main/java/kostka/moviecatalog/service/UserService.java

```diff
@@ -64,6 +67,14 @@ public User saveUser(final User user) {
         return userRepository.save(user);
     }
 
+    public List<UserListDto> getAllUsers() {
```

Comment:
Tip: If you have extra list dto for users, maybe you can have a look at JPA projections API ;) Another topic worth knowing.

Reply:
thanks for tip